tests/integration/test_logcollector/test_command_monitoring/test_command_execution.py

```diff
@@ -0,0 +1,214 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import pytest
+import sys
+from wazuh_testing.tools import monitoring
+from wazuh_testing.tools import get_service
+import wazuh_testing.logcollector as logcollector
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import LOG_COLLECTOR_DETECTOR_PREFIX, AGENT_DETECTOR_PREFIX
```

Comment:
```suggestion
from wazuh_testing.tools.monitoring import LOG_COLLECTOR_DETECTOR_PREFIX
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/1257/commits/1e33fb8c458aeb8a738e22706713ec98fbf25703